### DISCUSSION TITLE
inductor and transistor values

### DISCUSSION THREAD

**Discussion by timothymor** [micro-nova/AmpliPi#100] [Q&A] [answered]
would it be possible to provide the inductor and transistor values?

**Comment by timothymor** [comment]
I have highlighted the inductors and transistors without value .
[controller_board_pi - highlighted.pdf](https://github.com/micro-nova/AmpliPi/files/6267119/controller_board_pi.-.highlighted.pdf)
[preamp_board-highlighted.pdf](https://github.com/micro-nova/AmpliPi/files/6267120/preamp_board-highlighted.pdf)

**Comment by mabowers** [comment, marked as answer]
Hello @timothymor, thanks for your interest in AmpliPi. The schematics currently available on GitHub are for the AmpliPi prototype system. The finalized AmpliPi schematics will be posted shortly after the first batch of units ship from the Kickstarter (later this month).

**Reply by timothymor** [reply]
I already thought there were some mistakes
Like the r49 controller board should be 100k
I thought it was on purpose. 
Thanks for the update!

**Reply by linknum23** [reply]
@timothymor we still have not sent out the first developer units, once we do the updated schematics will be made available.

We really want to get the first batch exactly right and have our schematics match.

Thank you for your patience!
